docs/source/algorithms/DoublePulseFit-v1.rst

```diff
@@ -41,8 +41,37 @@ Usage
   y = y1+y2
   ws = CreateWorkspace(x,y)
 
+  # Create functions to fit
+  convolution =  FunctionFactory.createCompositeFunction('Convolution')
   innerFunction = FunctionFactory.createInitialized('name=GausOsc,A=0.2,Sigma=0.2,Frequency=1,Phi=0')
-  DoublePulseFit(Function=innerFunction, InputWorkspace=ws,  PulseOffset = delta, StartX=0.0, EndX=15.0, Output='DoublePulseFit')
+  deltaFunctions = FunctionFactory.createInitialized('(name=DeltaFunction,Height=1,Centre=-0.33,ties=(Height=1,Centre=-0.33);name=DeltaFunction,Height=1,Centre=0,ties=(Height=1,Centre=0))')
+  convolution.setAttributeValue('FixResolution', False)
+  convolution.add(innerFunction)
+  convolution.add(deltaFunctions)
+  innerFunctionSingle = FunctionFactory.createInitialized('name=GausOsc,A=0.2,Sigma=0.2,Frequency=1,Phi=0')
+  
+  DoublePulseFit(Function=innerFunctionSingle, InputWorkspace=ws,  PulseOffset = delta, StartX=0.0, EndX=15.0, Output='DoublePulseFit')
+  Fit(Function=convolution, InputWorkspace=ws, CreateOutput = True, StartX=0.0, EndX=15.0, Output='Fit', MaxIterations=1)
+
+  double_parameter_workspace = AnalysisDataService.retrieve('DoublePulseFit_Parameters')
+  double_col_values = double_parameter_workspace.column(1)
+
+  single_parameter_workspace = AnalysisDataService.retrieve('DoublePulseFit_Parameters')
```

Comment:
we are comparing the same table with itself. I have just tried running this and I get similar frequency values but not A

Reply:
ah, just saw the late one.

Reply:
```suggestion
  single_parameter_workspace = AnalysisDataService.retrieve('Fit_Parameters')
```